openml/tasks/functions.py

```diff
@@ -176,7 +176,11 @@ def _list_tasks(api_call):
                     'status': task_['oml:status']}
 
             # Other task inputs
-            for input in task_.get('oml:input', list()):
+            task_inputs = task_.get('oml:input')
```

Comment:
I'd prefer if you use the [force_list](https://github.com/openml/openml-python/blob/develop/openml/tasks/functions.py#L145) argument in order to avoid this if statement.

Reply:
Done.